pkgs/development/python-modules/misaka/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, fetchPypi, buildPythonPackage, cffi }:
+buildPythonPackage rec {
+  pname = "misaka";
+  name = "${pname}-${version}";
+  version = "2.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1yqrq3a5rracirmvk52n28nn6ckdaz897gnigv89a9gmyn87sqw7";
+  };
+
+  propagatedBuildInputs = [ cffi ];
+
+  doCheck = false;
```

Comment:
Why? Add a comment.

Reply:
...I actually don't remember why I did this. It tests fine. Removing it now.

Reply:
well, [that](https://github.com/NixOS/nixpkgs/pull/34296/#issuecomment-378171946)'s why. re-disabling.